apps/condo/domains/ticket/hooks/useTicketPropertyHintTableColumns.tsx

```diff
@@ -12,11 +12,12 @@ import { IFilters } from '@condo/domains/contact/utils/helpers'
 import { getTicketPropertyHintAddressesRender } from '@condo/domains/ticket/utils/clientSchema/Renders'
 import { TicketPropertyHintProperty } from '@condo/domains/ticket/utils/clientSchema'
 import { TicketPropertyHint } from '@app/condo/schema'
-import { useTicketPropertyHintContent } from './useTicketPropertyHintContent'
+import { TicketPropertyHintContent } from '../components/TicketPropertyHintContent'
```

Comment:
I would prefer `@condo/domains` here, since we're trying to avoid relative imports with parents (`..`), if it's possible. 

But it's optional btw 😃